webpack.config.common.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+    resolve: {
+        extensions: ['.js', '.jsx']
+    },
+    entry: './src/scripts/main.js',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: '[name].js',
+    },
+    plugins: [
+        new webpack.HotModuleReplacementPlugin(),
+        new CleanWebpackPlugin(),
+        new HtmlWebpackPlugin({
+            template: 'src/index.html'
+        })
+    ],
+    module: {
+        rules: [
+            {
+                test: /\.m?jsx?$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: "babel-loader",
+                    options: {
+                        presets: ['@babel/preset-env', "@babel/preset-react"]
```

Comment:
just wondering why do we need to duplicate presets here and in babel.config.json ?